README.md

```diff
@@ -320,6 +322,85 @@ i18n.offChangeLocale(fn);
 i18n.onceChangeLocale(fn);
 ```
 
+## Pluralization
+
+With this package, you can translate with pluralization. To do that you need to use the pipe separator `|` in your locale translations, like in the example below:
+
+```yml
+_locale: 'en'
+phone: 'zero phones | one phone | two phones | {count} phones'
+```
+
+Then just add the `count` argument to the `getTranslation` options:
+
+```html
+<p>{i18n.getTranslation('phone', { count: 0 })}</p>
+<p>{i18n.getTranslation('phone', { count: 1 })}</p>
+<p>{i18n.getTranslation('phone', { count: 2 })}</p>
+<p>{i18n.getTranslation('phone', { count: 3 })}</p>
+<p>{i18n.getTranslation('phone', { count: 1000 })}</p>
+```
+
+Output:
+
+```html
+<p>zero phones</p>
+<p>one phone</p>
+<p>two phones</p>
+<p>3 phones</p>
+<p>1000 phones</p>
+```
+
+### Custom rules
+
+Some languages (e.g. from the Slavic group) may have more complicated pluralization rules. It is possible to define custom pluralization rules to handle such cases. To do that add the `pluralizationRules` property to `i18n.setOptions`:
+
+```js
+i18n.setOptions({
+  // Key - language to applye the rule for, e.g 'pl'
+  // Value - function that takes the count as argument and returns index of the correct pluralization form
+  pluralizationRules: {
+    pl: value => {
+      if (value === 0) return 0;
+      if (value === 1) return 1;
+      if (value < 5) return 2;
+      return 3;
+    },
+  },
+});
```

Comment:
```suggestion
  pluralizationRules: {
    pl: count => {
      if (count === 0) return 0;
      if (count === 1) return 1;
      if (count < 5) return 2;
      return 3;
    },
  },
});
```

Let's stick to one name.

Reply:
I'd go nuts with this example and implement it properly (the current version doesn't work for 22, for example). You can find it [here](https://www.learnpolishtoday.com/lessons/counting-things-what-case-to-use).

Reply:
Exactly. A mechanism similar to the following needs to be implemented to cover ALL numbers ending with e.g. 2, such as 22, 32, 102, 122, 1002, etc.

function num_word(value, words){  
	value = Math.abs(value) % 100; 
	var num = value % 10;
	if(value > 10 && value < 20) return words[2]; 
	if(num > 1 && num < 5) return words[1];
	if(num == 1) return words[0]; 
	return words[2];
}